### DISCUSSION TITLE
Just talking

### DISCUSSION THREAD

**Discussion by PonyLucky** [PonyLucky/Pony-s-Manga-Manager#9] [General]
@002love Just saw your account.

As it says 'depressed' I opened the discussion channel, alright let's talk here.

Do you prefer Manga, Manhwua (Korean) or Manhua (Chinese) ?

**Comment by 002love** [comment]
I think I like manga the most 😅

**Reply by PonyLucky** [reply]
Hum... What I love in manga is the level of completeness. Is the sense that manga is older than the others, therefore it is more mature when it comes to what should be in it and what shouldn't.

Although there are exceptions, in general new manga really take after their predecessors. For example my favorite manga was Death Note until I got to know of Zankyou no Terror.

The first season of Death Note is dope though, after the death of L I had troubles viewing the rest. L was perfect to antagonize Light, they were so in sync ...
I did not find the balance between tension and understatement in season 2.

Still the first season was so far ahead in comparison to what I've seen that I could confidently say Death Node is my favorite.

---

As I said it before, until I got to know of Zankyou no Terror. For many points but let's keep it to two:
1. Stopped at the perfect moment, I remembered that the whole manga marked me to the point that I compared it to how much Hotaru no haka marked me (not the same emotion though... nor the same strength).
2. Plus the plot, as it is more realistic, we can feel better what the characters are going through.

What are your thoughts on it ? Do you feel the same way for another masterpiece ?

**Reply by 002love** [reply]
I think it's quite natural to experience excitement and excitement. As for me, it pumps your understanding of the whole point a lot and reading becomes more interesting.

I rather share the feelings of the characters themselves. Their sadness, feelings, excitement, joy, love, and so on. Because of this, after watching anime or reading manga, I can start to feel sad. 

Most likely, I will not be able to identify any specific works, the reading of which made a greater impression on me than viewing.

**Reply by PonyLucky** [reply]
I see, that is something I didn't get to experience.

I usually watch the anime before digging into manga if I want to have more. As manga is more detailed, you get more of what you like.

As time goes by I got to be picky in what I prefer:
- **Don't care** too much about the **setting** (like isekai, honestly it's not realistic).
- But the **credibility** of the plot afterward is important.
- I prefer **smart characters** (enough IQ to don't be surprised when anything happens to their life). With a plus if there are able to predict to some extent.
- I like shonen, so the world need to be harsh enough for the character to improve while maintaining credibility.
- Love when the drawing is beautiful *like many* :smile_cat: 

I don't need to check all of those to like a manga, so more like a dreamlist than requirements.

Even though I love the isekai lineup there are (for me) too much isekais where the main character is dump / or the world too light, like a perfect world.

I find the Korean's manhwua more harsh in this specific aspect. That's why as of today I enjoy more recent korean manhwua than manga. But Golgo 13 and Detective Conan kept their place in my heart.

Funny fact: When I went in Japan, my first wish was to buy the first tome of Golgo 13.

**Reply by PonyLucky** [reply]
What are your preferences ?

**Reply by 002love** [reply]
My preferences are probably pretty similar to yours. I don't even know what to say. 

If I really look by genre, it's probably action, drama, romance, science fiction. Again, I can't say anything about manga 😅.

To be honest I'm a little embarrassed that you write such a big comment, you try to make a point and I don't do that. In fact, I try to think of something, but nothing comes to mind....

**Reply by 002love** [reply]
By the way, this is really the first time I've ever talked to anyone on github. Not in discord and the like, but in github. I can say that communication here is quite unaccustomed, but at the same time, the discussions are built perfectly.  

Do you have any social networking accounts? You said above that you're pretty social. 
If this question made you uncomfortable, you don't have to answer it 😅

**Reply by PonyLucky** [reply]
Please don't mind it, I don't dislike monologues :laughing: 

*answer to first reply*

**Reply by PonyLucky** [reply]
I agree, first too. 

You talked about discord so I assume you have an account on it. I'll try to send you a private message on github, if I didn't assume right I'll send you my contacts on the social network of your choice.

It did not make me uncomfortable at all. thank you for caring.

*answer to second reply*

**Reply by PonyLucky** [reply]
Right, maybe it's possible to send private messages on GitHub but I don't see where.

It's not that important for Discord, as it is quite a private social network.

Here is my id: `ponylucky`, same picture... I made it myself and quite fond of it ;)

Please, add me as a friend, so we can communicate better.

**Reply by 002love** [reply]
Yes, thank you. I've sent you a friend request. I'm on Discord more often than github, so I don't think I'll take long to reply.

**Comment by 002love** [comment]
I'm really very pleased that you decided to talk to me, Mr. PonyLucky

**Reply by PonyLucky** [reply]
My pleasure, I am quite an antisocial therefore apart from my family (and job) I talk to only one person.
That's why I think I needed to talk to someone else :smile: 

Beneficial for both of us

**Comment by 002love** [comment]
By the way, I am relatively new to github issues, so I will ask you one question: 
Is it better to comment on all discussions or write a response to an existing comment?

**Reply by PonyLucky** [reply]
I think it is better to reply (respond to an existing comment) when you are directly answering and adding a comment otherwise.

As I didn't saw this specific case in the GitHub's guidelines it's up to interpretation.
